procs/RV32IM_3stage/CoreStates.bsv

```diff
@@ -0,0 +1,18 @@
+import RVTypes::*;
```

Comment:
Can you add a license to the top of each new file? Just copy the license from Core.bsv and write 2017 as the copyright year.